lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/BuiltinType.java

```diff
@@ -0,0 +1,19 @@
+package org.enso.interpreter.dsl;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/** An annotation denoting a node that should be wrapped for standard library export. */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.SOURCE)
+public @interface BuiltinType {
+  /**
+   * Comma-separated list of parameters of builting type
+   *
+   * @return list of params
+   */
+  // FIXME:
```

Comment:
what is this FIXME about? Is it still relevant? I think we should not merge FIXME into develop and if we have to, we should at least have an associated Pivotal issue for it.

Reply:
FIXIT – make it a real array :)

Reply:
Omission on my side. So the reason it is a string rather than an array is primarily because of `default`. Can't give it an empty array as a default because it complaints that it is not a constant. 
The alternative is that we provide an empty array for most of those annotations.